discretize/mixins/omfModule.py

```diff
@@ -0,0 +1,115 @@
+"""
+A class for converting ``discretize`` meshes to OMF objects
+"""
+
+import omf
+import numpy as np
+
+
+def ravelDataArray(arr, mesh):
```

Comment:
for naming, what do you think of `ravel_data_array`? This would be a bit more in line with pep8
```suggestion
def ravel_data_array(arr, mesh):
```

Reply:
I prefer pep8 - I feel `discretize` is a bit mixed on pep8 (snake case) and camel case right now?

Reply:
yeah, `discretize` is a mixed bag at the moment, but we are trying to push towards pep8, so if newer contributions are pep8, I think that will help ease the transition.

Reply:
Then I'll go ahead and make all the `mixins` pep8 in this PR since there's already significant changes happening here